srcpkgs/j/INSTALL.msg

```diff
@@ -0,0 +1,8 @@
+The J CLI (jconsole) is provided as /usr/bin/jc.
```

Comment:
This file should be removed.  The use of INSTALL messages is reserved for when failing to act would lead to either system instability or a non-functioning package.